pkg/sql/compile/scope.go

```diff
@@ -51,10 +51,16 @@ func PrintScope(prefix []byte, ss []*Scope) {
 
 // Run read data from storage engine and run the instructions of scope.
 func (s *Scope) Run(c *Compile) (err error) {
-	p := pipeline.New(s.DataSource.Attributes, s.Instructions, s.Reg)
+	p := pipeline.New(s.DataSource.Attributes, s.Instructions, s.Reg, s.DataSource.ExternalParam)
```

Comment:
Why this parameter is needed

Reply:
I need this param to get the param like filepath

Reply:
> I need this param to get the param like filepath

Why not write an opeator to handle this?